sap/5.9/modules/ROOT/pages/sap-connector-reference.adoc

```diff
@@ -13,6 +13,7 @@ Starting with version 5.0, SAP Connector also supports connecting to an on-premi
 * The *Document Listener* and *Function Listener* sources do not support Mule transactions. Although the *Transactional action* and *Transaction type* parameters (on the *Advanced tab*) are available for both sources, the transactional feature is not implemented.
 * The reconnection strategy on SAP Listeners (*Document Listener* and *Function Listener*) relies exclusively on the JCo library, which means that the Mule reconnection mechanism has no impact on reconnections.
 * The JCO Server that runs in the background manages the reconnection when there is a connection issue. The JCo RFC provider recognizes a communication error between the Java server and the ABAP gateway and attempts to reconnect. If the first reconnection fails, the next reconnection attempt occurs after waiting for one second. If the reconnection fails again, the reconnection wait time doubles from one second to two seconds, until eventually it reaches the default maximum waiting period of 3600 seconds, or one hour. Set the maximum reconnection timeout using the `-Djco.server.max_startup_delay=<reconnect delay in seconds>` JMV parameter.
+* The `XSTRING` data format in SAP is a binary blop format. To submit values in an `XSTRING` field, you must use the xref:dataweave::dw-numbers-functions-tohex.adoc[toHex] function to base 32 encode the binary within the XML submitted to the RFC step.
```

Comment:
```suggestion
* The `XSTRING` data format in SAP is a binary blop format. To submit values in an `XSTRING` field, you must use the xref:dataweave::dw-numbers-functions-tohex.adoc[toHex] function to base-32 encode the binary within the XML submitted to the RFC step.
```
Total nitpick. Should base-32 be hyphenated? Please disregard if not relevant! :)